ts/features/itwallet/common/components/ItwHighlightBanner.tsx

```diff
@@ -0,0 +1,330 @@
+  progress: SharedValue<number>;
+  width: number;
+  height: number;
+  rangeFactor: number;
+  reverse?: boolean;
+};
+
+const AnimatedLinearGradient = ({
+  progress,
+  width,
+  height,
+  rangeFactor,
+  reverse = false
+}: AnimatedLinearGradientProps) => {
+  const animationRange = useMemo(
+    () => width * rangeFactor,
+    [width, rangeFactor]
+  );
+
+  const animatedStart = useDerivedValue(() => {
+    const startX = interpolate(
+      progress.value,
+      reverse ? [1, 0] : [0, 1],
+      [-animationRange, animationRange],
+      Extrapolation.CLAMP
+    );
+    return vec(startX, height);
+  }, [height]);
+
+  const animatedEnd = useDerivedValue(() => {
+    const endX = interpolate(
+      progress.value,
+      reverse ? [1, 0] : [0, 1],
+      [width - animationRange, width + animationRange],
+      Extrapolation.CLAMP
+    );
+    return vec(endX, 0);
+  }, [width]);
+
+  return (
+    <LinearGradient
+      start={animatedStart}
+      end={animatedEnd}
+      mode="repeat"
+      colors={[
+        "#0B3EE3",
+        "#234FFF",
+        "#436FFF",
+        "#2F5EFF",
+        "#1E53FF",
+        "#1848F0",
+        "#0B3EE3",
+        "#1F4DFF",
+        "#2A5CFF",
+        "#1943E8",
+        "#0B3EE3"
+      ]}
+    />
+  );
+};
+
+const StaticContent = (props: Props) => {
+  const {
+    title,
+    description,
+    action,
+    onPress,
+    size = "large",
+    accessibilityHint,
+    accessibilityLabel,
+    accessibilityRole
+  } = props;
+
+  /* Generates a complete fallbackAccessibilityLabel by concatenating the title, content, and action
+   if they are present. */
+  const fallbackAccessibilityLabel = [title, description, action]
+    .filter(Boolean)
+    .join(" ");
+
+  const markdownRules = useMemo(
+    () => ({
+      Paragraph(paragraph: TxtParagraphNode, render: Renderer) {
+        return (
+          <LabelMini
+            key={getTxtNodeKey(paragraph)}
+            color="white"
+            weight="Regular"
+          >
+            {paragraph.children.map(render)}
+          </LabelMini>
+        );
+      },
+      Strong(strong: TxtStrongNode, render: Renderer) {
+        return (
+          <LabelMini
+            key={getTxtNodeKey(strong)}
+            color="white"
+            weight="Semibold"
+          >
+            {strong.children.map(render)}
+          </LabelMini>
+        );
+      }
+    }),
+    []
+  );
+
+  return (
+    <View
+      style={{ gap: 16 }}
+      accessible={true}
+      // A11y related props
+      accessibilityLabel={accessibilityLabel ?? fallbackAccessibilityLabel}
+      accessibilityHint={accessibilityHint}
+      accessibilityRole={action !== undefined ? accessibilityRole : "text"}
+    >
+      <HStack space={32}>
+        <VStack space={8} style={{ flex: 1 }}>
+          <View style={{ flexDirection: "row" }}>
+            <View style={{ alignSelf: "center" }}>
+              <Badge variant="highlight" text="Novità" />
```

Comment:
The text here is hardcoded. Is it intentional?

Reply:
Fixed here: https://github.com/pagopa/io-app/pull/6921/commits/c03fe9aa042279ece0e8e61d2ced234eb6b0746b
Thanks!